src/WalletInit/WalletForm.tsx

```diff
@@ -127,21 +127,6 @@ const PasswordInput = TextInput
 const PasswordConfirmationInput = TextInput
 const Actions = (props) => <View {...props} style={styles.actions} />
 
-const styles = StyleSheet.create({
-  safeAreaView: {
-    backgroundColor: COLORS.BACKGROUND,
```

Comment:
Removed background color from here and moved there -> https://github.com/Emurgo/yoroi-mobile/pull/2210/files#diff-48e01e998fe6260fc506d78e2ed55c16d489f21c4f8876b455bf9f244e422282R63